src/components/Navigator.js

```diff
@@ -90,6 +120,12 @@ const DEFAULT_STATE = {
   renaming: NO_RENAMING_STATE,
   filesList: {},
   nextNewFileIds: null,
+  selectedFileId: null,
+  selectedViewId: null,
+  selectedFileOpen: false,
```

Comment:
Why did we add this state - it should already be in the currentOpenFIleId props state. Now there's a possibility of inconsistency. We should be using the redux state since it should be the source of truth for what the state of which file is open.